### ISSUE TITLE
Can this framework be used for full supervision

### ISSUE THREAD

**Issue by tanjia123456** [gyshgx868/graph-ter#4]
hello, I have found many framework is Semi-supervised or unsupervised？ For node classification, all my data has labels, so semi-supervised or unsupervised is not applicable. Do you know of a suitable framework for my data set?

**Comment by gyshgx868** [comment]
Sorry for my late reply.

If you want to apply this framework to fully supervised tasks, you can simply add a cross-entropy loss at the following line:

https://github.com/gyshgx868/graph-ter/blob/3b9bda527a6a9559be835c5b84e6491ac8c5aa30/graph_ter_cls/runner/backbone_runner.py#L84

where you should also read the labels from the DataLoader.
